src/components/invite/Invite.js

```diff
@@ -147,6 +147,15 @@ const ShareBox = ({ level }) => {
 
   const share = useMemo(() => generateShareObject(shareTitle, shareMessage, shareUrl), [shareUrl])
 
+  const showShareIcons = useMemo(
```

Comment:
good design is when we don't even include library if it's not used on web or native
so please
1. create ShareIcons.web and .native
2. .native should export empty component e.g. `export default () => null`
3. .web should export the current ShareIcons implementation